src/user/models/kyc/kyc.service.spec.ts

```diff
@@ -0,0 +1,270 @@
+import { createMock } from "@golevelup/ts-jest";
+import { NotFoundException } from "@nestjs/common";
+import { Test, TestingModule } from "@nestjs/testing";
+import { Country } from "src/shared/models/country/country.entity";
+import { CountryService } from "src/shared/models/country/country.service";
+import { TestUtil } from "src/shared/test.util";
+import { SpiderSyncService } from "src/user/services/spider/spider-sync.service";
+import { SpiderService } from "src/user/services/spider/spider.service";
+import { AccountType } from "../user-data/account-type.enum";
+import { KycState, KycStatus, UserData } from "../user-data/user-data.entity";
+import { UserDataRepository } from "../user-data/user-data.repository";
+import { UserDataService } from "../user-data/user-data.service";
+import { KycUserDataDto } from "./dto/kyc-user-data.dto";
+import { KycProcessService } from "./kyc-process.service";
+import { KycInfo, KycService } from "./kyc.service"
+
+enum MockData {
+  CLEAN_DB,
+  EMPTY,
+  COMPLETE,
+  STARTED,
+}
+
+describe('KycService', () => {
```

Comment:
Didn't have a close look at the tests yet. There is already some UserData mock logic (src/user/models/user-data/__tests__/mock/user-data.entity.mock.ts). Would it be possible to reuse it?

Reply:
Didn't know that there is already something. I will check